core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml

```diff
@@ -7847,4 +7847,92 @@
          </categoryQuery>
       </category>
    </extension>
+   <extension
+         id="x"
+         point="org.polarsys.capella.common.ui.toolkit.browser.contentProviderCategory">
+      <category
+            id="---------- FunctionalChain ----------">
+      </category>
+      <category
+            id="flatFunctions"
+            isTopLevel="true"
+            name="Flat Functions">
```

Comment:
Can you name it "Involved Functions", for SystemAnalsis/Logical/Physical, and another query named "Involved Operational Activities" for OperationalAnalysis level